arelle/DisclosureSystem.py

```diff
@@ -335,6 +341,45 @@ def loadStandardTaxonomiesDict(self):
                                                  level=logging.ERROR)
                 etree.clear_error_log()
 
+    def loadValidTaxonomiesDict(self):
```

Comment:
Can't there be one method used for both loadStandardTaxonomiesDict and loadValidTaxonomiesDict, passing in the resulting output dict or something unduplicative?

Reply:
Agree that this is duplicative. Will look to see how we can remove the duplicity in a follow up PR.